rlclientlib/serialization/json_serializer.h

```diff
@@ -117,6 +117,8 @@ namespace reinforcement_learning { namespace logger {
       return error_code::success;
     }
 
+    int finalize(api_status* status, uint64_t original_event_count) { return finalize(status); }
```

Comment:
we are not passing in original_event_count to finalize here?

Reply:
original event count is required only in case of Protocol V2 and Interaction HTTP API sender which uses FB serializer in remaining cases we don't need them